.gitignore

```diff
@@ -17,3 +17,4 @@ target/
 .checkstyle
 .sonar/
 *.userprefs
+*.iml
```

Comment:
When you want to exclude .iml files, why are you commiting one?? Soapclient.iml !!!

Reply:
I was thinking git will autoexclude it, since it is in the .gitignore